lib/utils/note-utils.test.js

```diff
@@ -55,13 +50,13 @@ describe('noteTitleAndPreview', () => {
 
   it('should have enough text for an Expanded preview, even if the title is very long', () => {
     // Longer than the char limits
-    const title = 'foo'.repeat(titleCharacterLimit);
-    const paragraph = 'foo'.repeat(previewCharacterLimit);
+    const title = 'foo'.repeat(64);
+    const paragraph = 'foo'.repeat(200);
```

Comment:
the fact that these numbers match our lengths makes me worry that someone will assume there is more connection here than there is.

how about something like `.repeat( 500 )` or an otherwise big number

```js
const title = 'A really long title'.repeat( 100 );
const paragraph = 'A really long paragraph'.repeat( 100 );
```

Reply:
For sure